src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -264,9 +264,8 @@ void ResumeCtrlImpl::ApplicationResumptiOnTimer() {
       SDL_LOG_ERROR("Invalid app_id = " << *it);
       continue;
     }
-    if (!StartAppHmiStateResumption(app)) {
-      app->set_is_resuming(false);
-    }
+    StartAppHmiStateResumption(app);
```

Comment:
@ychernysheva consider changing `bool` result of that function to `void`

Reply:
Done in 0fa3d9f